osu.Game/Screens/Multiplayer/DrawableRoom.cs

```diff
@@ -0,0 +1,249 @@
+                                        rankBounds = new OsuSpriteText
+                                        {
+                                            Anchor = Anchor.CentreRight,
+                                            Origin = Anchor.CentreRight,
+                                            Text = "#0 - #0",
+                                            TextSize = 14,
+                                            Margin = new MarginPadding { Right = 10 },
+                                        },
+                                    },
+                                },
+                            },
+                        },
+                        new FillFlowContainer
+                        {
+                            Anchor = Anchor.BottomLeft,
+                            Origin = Anchor.BottomLeft,
+                            RelativeSizeAxes = Axes.X,
+                            AutoSizeAxes = Axes.Y,
+                            Direction = FillDirection.Vertical,
+                            Margin = new MarginPadding { Bottom = content_padding },
+                            Children = new Drawable[]
+                            {
+                                status = new OsuSpriteText
+                                {
+                                    TextSize = 14,
+                                    Font = @"Exo2.0-Bold",
+                                },
+                                beatmapInfoFlow = new FillFlowContainer<OsuSpriteText>
+                                {
+                                    RelativeSizeAxes = Axes.X,
+                                    AutoSizeAxes = Axes.Y,
+                                    Direction = FillDirection.Horizontal,
+                                    Children = new[]
+                                    {
+                                        beatmapTitle = new OsuSpriteText
+                                        {
+                                            TextSize = 14,
+                                            Font = @"Exo2.0-BoldItalic",
+                                        },
+                                        beatmapDash = new OsuSpriteText
+                                        {
+                                            TextSize = 14,
+                                            Font = @"Exo2.0-RegularItalic",
+                                        },
+                                        beatmapArtist = new OsuSpriteText
+                                        {
+                                            TextSize = 14,
+                                            Font = @"Exo2.0-RegularItalic",
+                                        },
+                                    },
+                                },
+                            },
+                        },
+                    },
+                },
+            };
+
+            Room.Name.ValueChanged += displayName;

```

Comment:
Directly binding to bindables of other objects is generally frowned upon. For now it should be okay because this is a view of a model, but at very least you need to unbind these events in a dispose override